api/lib/stores/block-store.js

```diff
@@ -102,10 +102,11 @@ class BlockStore {
   // this adds all the validator addresses to the database so we can easily check in the database which ones have an image and which ones don't
   async updateDBValidatorProfiles(validators) {
     const validatorRows = Object.values(validators).map(
-      ({ operatorAddress, name, identity }) => ({
+      ({ operatorAddress, name, identity, picture }) => ({
         operator_address: operatorAddress,
         name,
-        profile_identifier: identity
+        profile_identifier: identity,
+        picture
```

Comment:
why is this fixing it?

Reply:
Wouldn't this just overwrite the pictures in the db?

Reply:
https://github.com/luniehq/lunie/issues/5074#issuecomment-707924607

Reply:
Yes, it overwrites `null`s and outdated pictures with the current ones, which is good